### PR TITLE
File Added : Project Documentation & Assignment 5 and 6

### PR DESCRIPTION
**Pull request by urmilagagare**

### CODE REVIEW

File: urmila/HTML/Assignment6/Q1.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    border-top-right-radius: 10px;
+    display: flex;
+    justify-content: flex-end;
+}
+h1 {
+    margin-bottom: 50px;
+}
+label {
+    font-weight: bold;
+}
```
Comment (nileshp1122) [inline urmila/HTML/Assignment6/Q1.css]: add new line end of code

File: urmila/HTML/Assignment6/Q1.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+h1,
```
Comment (nileshp1122) [inline urmila/HTML/Assignment6/Q1.css]: h1 and h2 in same line     ex:     h1 , h2 {

File: urmila/HTML/Assignment5/style.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    padding: 0;
+}
+#background {
+    background-image: url(../background_image.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100vw;
+    height: 100vh;
+}
+p,
```
Comment (nileshp1122) [inline urmila/HTML/Assignment5/style.css]: arrange in same line like:  p, ul , h1 {

File: urmila/HTML/Assignment5/gallery.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    <body>
+        <div id="main">
+            <h1>Photo Gallery</h1>
+            <div class="img-container">
+                <img class="image" src="https://cdn.pixabay.com/photo/2020/09/07/17/40/birds-5552482_1280.jpg" alt="birds" />
+                <img class="image" src="https://cdn.pixabay.com/photo/2021/04/11/08/57/donkey-6169088_1280.jpg" alt="donkey" />
+                <img class="image" src="https://cdn.pixabay.com/photo/2021/04/12/17/53/flower-6173483_1280.jpg" alt="flower" />
+                <img class="image" src="https://cdn.pixabay.com/photo/2021/04/05/17/15/leaf-6154067_1280.jpg" alt="leaf" />
+                <img class="image" src="https://cdn.pixabay.com/photo/2021/04/15/15/04/desert-6181352_1280.jpg" alt="desert" />
+
```
Comment (nileshp1122) [inline urmila/HTML/Assignment5/gallery.html]: remove extra line

File: urmila/HTML/Assignment5/Resume.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                    <dd>-NEW English School & College Nagar</dd>
+                    <br />
+                    <dt>Graduation</dt>
+                    <dd>-MIT College in Pune</dd>
+                    <br />
+                </dl>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (nileshp1122) [inline urmila/HTML/Assignment5/Resume.html]: add line end of code

File: urmila/HTML/Assignment5/Home.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                    <li><a href="./A5_resume.html" target="_blank">Resume</a></li>
+                    <li><a href="./A5_hobbies.html" target="_blank">My Hobbies</a></li>
+                    <li>
+                        <a href="./A5_gallery.html" target="_blank">Gallery</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (nileshp1122) [inline urmila/HTML/Assignment5/Home.html]: add line end of code

**Review by nileshp1122 — CHANGES_REQUESTED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by nileshp1122 — APPROVED** [review]

**Review by Ashwin2812 — APPROVED** [review]

File: urmila/HTML/Assignment6/Q2.css [hunk 7]
```diff
@@ -0,0 +1,57 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+h1,
+h2 {
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment6/Q2.css]: h1 nd h2 should be in single line

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

File: urmila/HTML/Assignment6/Q2.css [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+#other {
+    margin: 20px 0;
+}
+.container {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 70%;
+}
+
```
Comment (shubhamdalal49) [inline urmila/HTML/Assignment6/Q2.css]: remove this extra line

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

File: urmila/HTML/Assignment5/style.css [hunk 9]
```diff
@@ -0,0 +1,31 @@
+ {
```
Comment (Pawan1005) [inline urmila/HTML/Assignment5/style.css]: tag missing here

File: urmila/HTML/Assignment5/Hobbies.html [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+                        <h2><img src="./movies_icon.png" alt="Watching" width="20px" /> Movies</h2>
+                    </li>
+                    <li>
+                        <h2><img src="./game_icon.png" alt="Playing" width="20px" /> Gaming</h2>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline urmila/HTML/Assignment5/Hobbies.html:30]: make 1 indentation for li tag

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**